semcore/side-panel/src/SidePanel.jsx

```diff
@@ -148,6 +164,14 @@ function Panel(props) {
 
   useFocusLock(sidebarRef, true, 'auto', !visible, true);
 
+  const hasLabel = Boolean(props['aria-label'] || props['aria-labelledby']);
+
+  logger.warn(
+    !hasLabel,
+    "'aria-label' or 'aria-labelledby' are required for SidePanel component",
+    props['data-ui-name'] || Panel.displayName,
+  );
```

Comment:
what do you think about adding required aria-* props to types?

Reply:
I would prefer to keep types flat